### ISSUE TITLE
work without socket.io ?

### ISSUE THREAD

**Issue by DragorWW** [thoov/ember-websockets#43]
How to use this addon without socket.io
``` 
$ ember server

version: 0.2.7
Error reading socket.io file. Please run `ember g socket-io` first
Error: Error reading socket.io file. Please run `ember g socket-io` first
    at Class.module.exports.included ([project]/node_modules/ember-websockets/index.js:24:13)
    at EmberApp.<anonymous> ([project]/node_modules/ember-cli/lib/broccoli/ember-app.js:312:15)
    at Array.filter (native)
    at EmberApp._notifyAddonIncluded ([project]/node_modules/ember-cli/lib/broccoli/ember-app.js:307:45)
    at new EmberApp ([project]/node_modules/ember-cli/lib/broccoli/ember-app.js:103:8)
    at Object.<anonymous> ([project]/Brocfile.js:5:11)
    at Module._compile (module.js:460:26)
    at Object.Module._extensions..js (module.js:478:10)
    at Module.load (module.js:355:32)
    at Function.Module._load (module.js:310:12)
```

**Comment by thoov** [comment]
@DragorWW thanks for finding this issue. I believe I have fixed it in the latest version (1.2.2) so if you could install that version you should fix your problem.

**Comment by DragorWW** [comment]
I have tested all is well.
